back/lib/voting_method/single_voting.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module VotingMethod
+  class SingleVoting < Voting
+    def initialize(participation_context)
+      participation_context.voting_max_votes_per_idea = 1
```

Comment:
This assumes that the voting method is always consulted when creating a participation context, right before saving. I think it would be better to use `assign_defaults_for_participation_context` or something analogous as a more consistent and robust alternative. We should also add a validation.